util/quality-measures-additional-info.json

```diff
@@ -0,0 +1,1217 @@
+[
+  {
```

Comment:
The descriptions are programmatically generated from the pdfs, but the shortened names (and things like the algorithm used to calculate them) are manually changed. 👍 on a more descriptive name.